src/main/main.cpp

```diff
@@ -122,14 +122,14 @@ usage(int err = 1)
           "      --base64             Use base64 for --printtxn and --signtxn\n"
           "      --catchup-at SEQ     Do a catchup at ledger SEQ, then quit\n"
           "                           Use current as SEQ to catchup to "
-          "'current'"
```

Comment:
clang-format is the one that split the lines randomly like this, you can safely join this line and the next one while you're making this change

Reply:
Ah. Yep done.